### PR TITLE
Release V1.2.0

### PR DESCRIPTION
**Pull request by Murmele**
-  #266 
- #245
- #271 
- #273
- #281
- #283
- #280 
- #285 
- #289 
- #286
- #287 
- #295 
- #298 
- #302
- #312
- #317 
- #314 
- #325 
I think we can increase the Minor number, because we added also some bigger features

TODO: update amend dialog image

### CODE REVIEW

File: docs/changelog.md [hunk 1]
```diff
@@ -1,3 +1,33 @@
+### v1.2.0 - 2022-XX-XX
+
+Bug fix release
+
+#### Added
+* Add support for solving merge conflicts for whole files
+* Support rebasing with conflict solving
+
```
Comment (Murmele) [inline docs/changelog.md:8]:
```suggestion
* implement amending commits (author, committer, commit message, adding tree to commit)

```

**Comment by Murmele** [comment]
@exactly-one-kas do you have anything against releasing on sunday?

**Comment by exactly-one-kas** [comment]
Is #276 still relevant? If so, I'd recommend waiting for the fix

**Comment by Murmele** [comment]
> Is #276 still relevant? If so, I'd recommend waiting for the fix

Yes it is relevant. Lets check for that

**Comment by Murmele** [comment]
@exactly-one-kas I would propose moving the release date

**Comment by Murmele** [comment]
@exactly-one-kas just add things to changelog if I forgott something. 
Perhabs I am not familar where ifacer.py is used :/ Do you know why we need it and where it is executed?

**Comment by exactly-one-kas** [comment]
> @exactly-one-kas I would propose moving the release date

Yeah, I don't think we'll be able to make it today

> Perhabs I am not familar where ifacer.py is used :/ Do you know why we need it and where it is executed?

I think this is used to regenerate the Scintilla interface files when they become incompatible due to an update

**Comment by Murmele** [comment]
> 

Ah ok. So it is not that important, because it will be executed only once per scintilla update?

**Comment by exactly-one-kas** [comment]
It has to be run manually

**Comment by Murmele** [comment]
> 

Ah ok. Then I delete it from the changelog, because it is not relevant for the user

File: docs/changelog.md [hunk 2]
```diff
@@ -1,52 +1,87 @@
...
+#### Changed
+* Fix Segmentation fault when using space to stage files
+* Fix menubar color in dark theme
+* Filter only branches, tags, remotes attached to selected commit
+* Fix crash when global GIT config is invalid
+* Fix crash when having errors while adding a remote account
+* Fix updater on windows, macos and linux (flatpak)
+* Instead of using a soft reset instead of amend, show a dialog to change commit properties and amend changes then
+* Fix discarding file leading to discarding submodule changes
+* Fix rebase log messages during rebase
```
Comment (Murmele) [inline docs/changelog.md:22]:
```suggestion
* Fix rebase log messages during rebase
* Improve SSH config handling
```

**Comment by Murmele** [comment]
It fails because the timestamp of the changelog is in the future. I set the release date a few days ago to tomorrow.
@exactly-one-kas can you add information about your mr #302 to the changelog?

**Comment by exactly-one-kas** [comment]
I'd like to pitch #314 as the last blocking feature - it would allow package maintainers (with the focus being on the AUR) to build the package as-is instead of being required to patch it a lot

**Comment by Murmele** [comment]
> I'd like to pitch #314 as the last blocking feature - it would allow package maintainers (with the focus being on the AUR) to build the package as-is instead of being required to patch it a lot

Ok I added it. I will review it later

**Comment by exactly-one-kas** [comment]
I'm not sure if #315 makes sense here, since it is a *lot* of work and some redesigns

**Comment by Murmele** [comment]
> 

Great work with this issue! But yeah it sounds like a big feature. I think for this feature we have enough. So the user get the automatic download feature ready which is on my opinion a big issue when updating?

**Comment by exactly-one-kas** [comment]
> automatic download feature

Are you talking about #245? That's already done and merged

**Comment by Murmele** [comment]
> > automatic download feature
> 
> Are you talking about #245? That's already done and merged

yes exactly, but we never merged a backport and released a version with it. So I think we should try to get a new version as soon as possible.

**Comment by jonadem** [comment]
> By the way. Only one star is missing to reach 500 :D

Here you are ! :-D

File: README.md [hunk 3]
```diff
@@ -154,7 +154,7 @@ Install `gittyup-git` for the VCS build.
 
 **Homebrew**
 
-Install the `gittyup` cask from [Homebrew](https://brew.sh/).
+Install the `gittyup` cask from [Homebrew](https://formulae.brew.sh/cask/gittyup/).
```
Comment (Murmele) [inline README.md:157]:
```suggestion
Install the `gittyup` cask from [Homebrew](https://formulae.brew.sh/cask/gittyup).
```

File: docs/changelog.md [hunk 4]
```diff
@@ -1,52 +1,91 @@
...
+* Implement support for Gitea instances
+
+#### Changed
+* Fix Segmentation fault when using space to stage files
+* Fix menubar color in dark theme
+* Filter only branches, tags, remotes attached to selected commit
+* Fix crash when global GIT config is invalid
+* Fix crash when having errors while adding a remote account
+* Fix updater on windows, macos and linux (flatpak)
+* Instead of using a soft reset instead of amend, show a dialog to change commit properties and amend changes then
```
Comment (Murmele) [inline docs/changelog.md:21]:
```suggestion
```

**Comment by Murmele** [comment]
- Fix Arch linux build

**Comment by Murmele** [comment]
@exactly-one-kas from my side this is fine

**Comment by exactly-one-kas** [comment]
I'll just merge #327, after that we can release

**Comment by Murmele** [comment]
> 

Great :D Thank you! 🚀

**Review by exactly-one-kas — APPROVED** [review]

**Comment by Murmele** [comment]
crossing fingers that everything works fine ^^

**Pull request merged**